strongbox-db-schema/src/main/java/org/carlspring/strongbox/db/schema/Vertices.java

```diff
@@ -4,19 +4,30 @@
 {
 
     String ARTIFACT = "Artifact";
+
     String REMOTE_ARTIFACT = "RemoteArtifact";
+
     String GENERIC_ARTIFACT_COORDINATES = "GenericArtifactCoordinates";
+
     String ARTIFACT_COORDINATES = "ArtifactCoordinates";
-    
+
     String RAW_ARTIFACT_COORDINATES = "RawArtifactCoordinates";
+
     String MAVEN_ARTIFACT_COORDINATES = "MavenArtifactCoordinates";
+
     String NPM_ARTIFACT_COORDINATES = "NpmArtifactCoordinates";
+
     String NUGET_ARTIFACT_COORDINATES = "NugetArtifactCoordinates";
+
     String PYPI_ARTIFACT_COORDINATES = "PypiArtifactCoordinates";
+
     String RPM_ARTIFACT_COORDINATES = "RpmArtifactCoordinates";
-    
+
     String ARTIFACT_TAG = "ArtifactTag";
+
     String ARTIFACT_ID_GROUP = "ArtifactIdGroup";
-    
+
     String USER = "User";
+
+    String TRANSACTION_PROPAGATION = "TransactionPropagationTestVertex";
```

Comment:
@anki2189 AFAIK this is just for test, so we better use something else there and drop this `TransactionPropagationTestVertex` from schema.

Reply:
Changes done.